src/test/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAgeTest.java

```diff
@@ -70,4 +87,41 @@ public void testGetEndpoint() throws Exception {
         assertEquals(maxAge, endpoint.get(request, response));
     }
 
+     @Test
+    public void testGetEndpointWithCacheAPIEnabled() throws Exception {
+        try (MockedStatic<ApplicationConfiguration> mockedAppConfig = mockStatic(ApplicationConfiguration.class)) {
+            MockedStatic<HttpClients> mockedHttpClients = mockStatic(HttpClients.class);
+
+            ApplicationConfiguration appMockConfiguration = mock(ApplicationConfiguration.class);
+            Babbage babbageMock = mock(Babbage.class);
+
+            when(ApplicationConfiguration.appConfig()).thenReturn(appMockConfiguration);
+            when(appMockConfiguration.babbage()).thenReturn(babbageMock);
+            when(babbageMock.isLegacyCacheAPIEnabled()).thenReturn(true);
+            when(babbageMock.getLegacyCacheProxyUrl()).thenReturn("http://legacycache.example.com"); 
+
+            HttpServletRequest mockRequest = mock(HttpServletRequest.class);
+            when(mockRequest.getMethod()).thenReturn("GET");
+            when(mockRequest.getHeaderNames()).thenReturn(Collections.enumeration(Arrays.asList("Accept-Language", "User-Agent")));
+            when(mockRequest.getHeader("Accept-Language")).thenReturn("en-US");
+            when(mockRequest.getHeader("User-Agent")).thenReturn("Mozilla/5.0");
+
+            CloseableHttpClient mockHttpClient = mock(CloseableHttpClient.class);
+            when(HttpClients.createDefault()).thenReturn(mockHttpClient);
+
+            CloseableHttpResponse mockResponse = mock(CloseableHttpResponse.class);
+            RequestBuilder mockRequestBuilder = mock(RequestBuilder.class);
+            HttpUriRequest mockHttpUriRequest = mock(HttpUriRequest.class);
+
+            when(mockRequestBuilder.build()).thenReturn(mockHttpUriRequest);
+
+            when(mockHttpClient.execute(any())).thenReturn(mockResponse);
+            when(mockResponse.getFirstHeader("Cache-Control")).thenReturn(new BasicHeader("Cache-Control", "max-age=3600"));
+
+            int maxAge = endpoint.getMaxAge(mockRequest);
+            verify(mockHttpClient, times(1)).execute(any());
+
+            assertEquals(3600, maxAge);
+        }
+    }
```

Comment:
There are a couple more tests you could add here, for instance, what happens when `Cache-Control` is not set, or a test for when its value has multiple directives (e.g. `public,max-age=123`). The setup code for both is going to be quite long and very similar, so you can use nested classes (that's how they're called in JUnit 5, there must be an equivalent in JUnit 4).